doc/changelog/01-kernel/19262-template-no-sup-constraint-kernel.rst

```diff
@@ -0,0 +1,9 @@
+- **Changed:**
+  the kernel typing rules for template polymorphic inductive types do not
+  require anymore adding global constraints when applied enough. Rather,
+  template polymorphic inductive types are now a special kind of universe
+  polymorphic inductive types that do not need explicit instances and
+  can handle some amount of algebraic universe levels. The new rules are
```

Comment:
and Prop

Reply:
That's part of the "some", I think.

Reply:
Don't we usually say that Prop is not a universe level?

Reply:
That's true in the implementation, but the term "algebraic" is not even defined from the user side.